src/components/DataListSection.tsx

```diff
@@ -77,36 +77,49 @@ export const DataListSection = ({
     skipLastDivider: true
   });
 
+  if (listData?.length) {
+    return (
+      <View>
+        {!!sectionTitle && (
+          <SectionHeader onPress={navigate} title={sectionTitle ?? getTitleForQuery(query)} />
+        )}
+        {!!limit &&
+          (listData?.length ? (
```

Comment:
if we check for `listData?.length` in line 80 now, we do not need to do it here again, right?

Reply:
This js is used in more than one place. All pages have different props. And I tried to keep the latest updated version. But I can change it. 👍

Reply:
https://github.com/ikuseiGmbH/smart-village-app-app/pull/583/commits/c572a17d9f6c37319dec493afd2b8a6a9c43d631